flake.nix

```
@@ -0,0 +1,109 @@
+{
+  description = "Searx plugin for Searching the Green Web";
+
+  inputs.nixpkgs = {
+    type = "github";
+    owner = "NixOS";
+    repo = "nixpkgs";
+    ref = "21.05";
+  };
```

How about:
```suggestion
  inputs.nixpkgs.url = "github:NixOS/nixpkgs/21.05";
```
Or maybe even:
```suggestion
  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable-small";
```

---

With "nixos-unstable-small" something doesn't work during the plugin build, but I'm not too inclined to investigate, 
21.05 seems recent enough